service/frontend/protojson_marshaler.go

```diff
@@ -30,38 +30,70 @@ import (
 	"io"
 
 	"github.com/grpc-ecosystem/grpc-gateway/v2/runtime"
-	"google.golang.org/protobuf/encoding/protojson"
 	"google.golang.org/protobuf/proto"
+
+	commonpb "go.temporal.io/api/common/v1"
+	"go.temporal.io/api/temporalproto"
+)
+
+const (
+	mimeAll                   = "*"
+	mimePrettyJSONNoShorthand = "application/json+pretty+no-payload-shorthand"
+	mimeJSONNoShorthand       = "application/json+no-payload-shorthand"
+	mimePrettyJSONShorthand   = "application/json+pretty"
 )
```

Comment:
nit: I know we do this all over the codebase, but I prefer just having the value in line and avoid the extra lookup.

Reply:
I inlined them